### PR TITLE
SDU-1893: Added new inclusion

### PR DESCRIPTION
**Pull request by PaulinaDrozda**

### CODE REVIEW

**Review by eternalamenity — APPROVED** [review]

File: thing-management.md [hunk 1]
```diff
@@ -482,6 +482,63 @@ Message (command):
...
+    "val_t": "null",
+    "props": {},
+    "tags": null,
+    "uid": "123456789"
+}
+```
+
+### Inclusion v2 PIN requested
+
+If included device requests a PIN then the following message will sent:
```
Comment (boguslaw-wojcik) [inline thing-management.md:506]: will *be* sent

File: thing-management.md [hunk 2]
```diff
@@ -482,6 +482,63 @@ Message (command):
...
+
+Topic: `pt:j1/mt:evt/rt:ad/rn:zw/ad:1`
+
+Message (command):
```
Comment (boguslaw-wojcik) [inline thing-management.md:510]: It's not command it's event.

File: thing-management.md (hunk 1 above)
Comment (PaulinaDrozda) [reply thing-management.md]: Fixed

**Review by boguslaw-wojcik — APPROVED** [review]
One comment: I don't like a lack of some kind of session indicator in this process - if someone quits the process in the middle and restarts, it's not apparent if the pin will be delivered to the correct process. Maybe it's not a problem...

**Pull request merged**